### DISCUSSION TITLE
fusion future

### DISCUSSION THREAD

**Discussion by dos1973** [matt8707/ha-fusion#366] [General]
Hi,
To be honest i really regular check if there is an update available as i really like the project. 

I don't believe Matt just codes aimlessly... i think, he likely has a some target version in mind. Ideally, this version should be enriched and improved by the community with ideas. 

Therefore, I wanted to ask if there is a roadmap outlining what's coming and possibly a mockup of how a future dashboard could look like.

I would greatly appreciate it. Thanks

**Comment by matt8707** [comment]
There's no plan, really. I'm just implementing what I think are good ideas, and I do listen to feedback. I don't have a lot of spare time rn so development is slow but steady.
